docs/content/preview/admin/yb-admin.md

```diff
@@ -974,6 +975,52 @@ Tablet 1          e509cf8eedba410ba3b60c7e9138d479  e509cf8eedba410ba3b60c7e9138
 Snapshot          4963ed18fc1e4f1ba38c8fcf4058b295  4963ed18fc1e4f1ba38c8fcf4058b295
 ```
 
+#### import_snapshot_selective
+
+Imports only the specified tables from the specified snapshot metadata file.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    import_snapshot_selective <file_name> \
+    [<keyspace> <table_name> [<keyspace> <table_name>]...]
+```
+
+* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *file_name*: The name of the snapshot file to import
+* *keyspace*: The name of the database or keyspace
+* *table_name*: The name of the table
+
+{{< note title="Note" >}}
+
+The *keyspace* and the *table* can be different from the exported one.
```

Comment:
The keyspac can be different from the exported one. But name of the table needs to be the same.